### PR TITLE
Added pull endpoint

### PR DESCRIPTION
**Pull request by mrxbox98**

### CODE REVIEW

File: reference/Client-API.yaml [hunk 1]
```diff
@@ -2609,6 +2609,47 @@ paths:
...
         description: Write to file
+  '/servers/{server}/files/pull':
+    parameters:
+      - schema:
+          type: string
+        name: server
+        in: path
+        required: true
+    post:
+      summary: ''
```
Comment (TekExplorer) [inline reference/Client-API.yaml]: Ideally, fill out the summary to describe what the endpoint is. for instance: "Pull Remote File"

File: reference/Client-API.yaml [hunk 2]
```diff
@@ -2609,6 +2609,47 @@ paths:
...
+      operationId: post-servers-server-files-pull
+      responses:
+        '200':
+          description: OK
+      description: Download a file from an external URL
+      requestBody:
```
Comment (TekExplorer) [inline reference/Client-API.yaml:2626]: for this section, ideally include a description of the request body

File: reference/Client-API.yaml [hunk 3]
```diff
@@ -2609,6 +2609,47 @@ paths:
               Example 1:
                 value: eula=true
         description: Write to file
+  '/servers/{server}/files/pull':
```
Comment (TekExplorer) [inline reference/Client-API.yaml:2612]: Add the "Server Files" tag to this endpoint

File: reference/Client-API.yaml [hunk 4]
```diff
@@ -2609,6 +2609,47 @@ paths:
...
+      responses:
+        '200':
+          description: OK
+      description: Download a file from an external URL
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
```
Comment (TekExplorer) [inline reference/Client-API.yaml:2631]: ideally mark required elements, such as "url"

**Review by TekExplorer — CHANGES_REQUESTED** [review]
Not bad, just add some missing stuff and I'll merge it.
- Tag
- Summary/Title
- Description of the request body
- Required/Non-Null values in the request or response

**Comment by mrxbox98** [comment]
Issues fixed. 👍

**Review by TekExplorer — APPROVED** [review]
All good!

**Pull request merged**